src/integer.c

```diff
@@ -1112,114 +1108,115 @@ Obj FuncINT_STRING ( Obj self, Obj string )
 
 /****************************************************************************
 **
-*F  EqInt( <gmpL>, <gmpR> ) . . . . . . . . .  test if two integers are equal
+*F  EqInt( <opL>, <opR> ) . . . . . . . . . .  test if two integers are equal
 **
-**  
-**  'EqInt' returns 1  if  the two integer   arguments <intL> and  <intR> are
-**  equal and 0 otherwise.
+**  'EqInt' returns 1 if the two integer arguments <opL> and <opR> are equal
```

Comment:
Here the old comments did not even make sense, as they referred to outdated argument names.